reconcile/utils/ocm.py

```diff
@@ -320,6 +320,7 @@ def get_ocm_spec(
             provider=cluster["cloud_provider"]["id"],
             region=cluster["region"]["id"],
             channel=cluster["version"]["channel_group"],
+            # should this be version=cluster["version"]["raw_id"] as for OSD ?
```

Comment:
Yes. there are two fixtures with the payloads returned by OCM in the tests dir: https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/test/fixtures/clusters/rosa_spec.json#L156-L159

Reply:
I was just wondering if the ROSA versioning would  follow the same behavior on upgrade (`.openshift_version` getting bumped before the upgrade is actually finished, `.version.raw_id` getting bumped only once all nodes are upgraded)

Reply:
My guess is that it is the same process being applied, so I'll push the same setting on ROSA.

Reply:
9ce8dcb